raiden/ui/cli.py

```diff
@@ -914,18 +914,3 @@ def print_step(description: str, error: bool = False) -> None:
 
     if not success:
         sys.exit(1)
-
-
-@run.command(
-    help=(
-        "Start an echo node.\n"
-        "Mainly useful for development.\n"
-        "See: https://raiden-network.readthedocs.io/en/stable/api_walkthrough.html"
-        "#interacting-with-the-raiden-echo-node"
```

Comment:
Can you update the docs as well? Makes probably the most sense to move these docs to the new echo node.